SR2019/2019-04-23-competition-wrap-up.md

```diff
@@ -0,0 +1,52 @@
+---
+to: Student Robotics 2019
+subject: Final competition details
+---
+
+![Competition Photo](https://studentrobotics.org/images/content/news/sr2019/sr2019-photo.jpg)
+
+Hi all,
+
+First off, we'd like to thank you for taking part in Student Robotics 2019 and making it such a successful event! An article about the competition, results, and game has been [posted on our website](https://studentrobotics.org/news/2019-04-09-sr2019-tlc-crowned-champion/)
+
+Congratulation again to our winners:
+
+|        Prize          |            Team
+|-----------------------|-----------------------------------------------
+| 1st Place             | [The Ladies' College Guernsey](http://www.ladiescollege.com/)
+| 2nd Place             | [Hills Road Sixth Form College](http://www.hillsroad.ac.uk/)
+| 3rd Place             | [Hampton School](https://hamptonschool.org.uk/) and [Lady Eleanor Holles School](https://www.lehs.org.uk/)
```

Comment:
Maybe co-ordinate with @antoinepetty on how to make it clearer that this _isn't_ a tied position?

Reply:
I just copied the table exactly from the website. I suspect putting in TLAs might help with this?

Reply:
I mostly agree on adding the TLAs -- that was my suggestion for the website. The minor gotcha is that "HAM: Hampton School and Lady Eleanor Holles School" could easily read as an accidentally omitted TLA.

The alternative I'd been wondering about was "HAM: <Hampton> and <Lady Eleanor Holles> Schools" or even "HAM: Lady Eleanor Holles School and Hampton School".
The latter has the benefit that it feels just a bit more likely to be clear that it's a joint team (since I'd hope people could work out that the TLA usually relates to the school name and that HAM is derived from Hampton), but that still doesn't feel like a perfect solution.

Reply:
I've updated the wording so that the photos get attributed back to the person who took them. (or at least, who uploaded them to Google). I suspect this is as good as we can get it.

Reply:
It's an interesting one. Seeing as it's currently live on the website as this, I think it's probably best to just leave this as-is. For the sake of this email, it's probably fine. If we want to get this as correct as possible, we should target thoughts towards the website instead.